packaging/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+if (WIN32) #Starting simple with archive only packages
```

Comment:
could we use a generator expression here? ([docs](https://cmake.org/cmake/help/latest/guide/tutorial/Adding%20Generator%20Expressions.html))

i.e.
```
set(msvc_cxx "$<COMPILE_LANG_AND_ID:CXX,MSVC>")
```